memorywipe.py

```diff
@@ -325,13 +326,13 @@ def _chk_compat_hdparm(self):
         
     def _chk_freeze(self):
         hdparm_output = subprocess.check_output(["sudo", "hdparm", "-I", f"{self.partition}"], text=True)
-        if "frozen" not in hdparm_output:
```

Comment:
I tried the piping of multiple grep statements you mentioned in slack, but it did not work. So, I have tried this alternative of searching for not frozen. Even though there are multiple white spaces between not and frozen, this approach is working fine.